qcelemental/molutil/align.py

```diff
@@ -263,6 +263,7 @@ def _plausible_atom_orderings_wrapper(
             print(atomfmt2.format(auniq[at][:6], *[ageom[at][i] - rgeom[at][i] for i in range(3)]))
 
     if do_plot:
+        # TODO Missing import
```

Comment:
yeah, this is a relic from when I was developing the code in jupyter and had the target, before, and after mols pop up for sanity checking. possibly this should be hidden further now.